railties/lib/rails/commands/credentials/credentials_command/diffing.rb

```diff
@@ -1,37 +1,42 @@
 # frozen_string_literal: true
 
 module Rails::Command::CredentialsCommand::Diffing # :nodoc:
+  GITATTRIBUTES_ENTRY = <<~END
+    config/credentials/*.yml.enc diff=rails_credentials
+    config/credentials.yml.enc diff=rails_credentials
+  END
+
   def enroll_project_in_credentials_diffing
-    if enrolled?
-      true
-    else
-      gitattributes.write(<<~end_of_template, mode: "a")
-        config/credentials/*.yml.enc diff=rails_credentials
-        config/credentials.yml.enc diff=rails_credentials
-      end_of_template
-
-      say "Project successfully enrolled!"
+    unless enrolled_in_credentials_diffing?
+      gitattributes.write(GITATTRIBUTES_ENTRY, mode: "a")
+
+      say "Enrolled project in credentials file diffing!"
```

Comment:
We are logging only on "success" here.  I feel like CLI commands usually either log only on failure, or log on both failure and success.

@kaspth I noticed you modified this to only log when enrolling (i.e. don't log when already enrolled).  What are your thoughts about logging on both (or only on failure)?

Reply:
I think it makes sense to log on both